resources/translations.mjs

```diff
@@ -101,8 +101,420 @@ export const todos = {
+        "一般的な木製キーボード",
+        "リサイクルされた木製チップ",
+        "信じられないほどのゴムチップ",
+        "人間工学に基づいた花崗岩のシャツ",
+        "おいしい冷凍キーボード",
+        "ゴージャスなスチールソープ",
+        "エレガントな冷凍自転車",
+        "リサイクルスチールチェア",
+        "モダンブロンズソーセージ",
+        "エレガントな木製チーズ",
+        "小さなプラスチックソーセージ",
+        "豪華な冷凍靴",
+        "なめらかなプラスチックソーセージ",
+        "手作りの新鮮なソーセージ",
+        "信じられないほどのソフトチェア",
+        "リサイクルされた木製石鹸",
+        "新鮮な魚をリサイクルしました",
+        "実用的なソフトグローブ",
+        "実用的なソフトチップ",
+        "小さな柔らかい鶏",
+        "インテリジェントな新鮮なマウス",
+        "現代の金属マウス",
+        "おいしい花崗岩の手袋",
+        "小さなスチールシャツ",
+        "洗練されたコンクリートコンピューター",
+        "洗練された冷凍シャツ",
+        "インテリジェントコンクリートシューズ",
+        "ゴージャスな木製のテーブル",
+        "一般的な木製キーボード",
+        "リサイクルされた木製チップ",
+        "信じられないほどのゴムチップ",
+        "人間工学に基づいた花崗岩のシャツ",
+        "おいしい冷凍キーボード",
+        "ゴージャスなスチールソープ",
+        "エレガントな冷凍自転車",
+        "リサイクルスチールチェア",
+        "モダンブロンズソーセージ",
+        "エレガントな木製チーズ",
+        "小さなプラスチックソーセージ",
+        "豪華な冷凍靴",
+        "なめらかなプラスチックソーセージ",
+        "手作りの新鮮なソーセージ",
+        "信じられないほどのソフトチェア",
+    ],
+    es: [
+        "Sombrero de granito electrónico",
+        "Mesa suave a medida",
+        "Bicicleta ergonómica fresca",
+        "Lujoso queso de madera",
+        "Hermosa pizza fresca",
+        "Camisa de goma rústica",
+        "Jabón de goma moderno",
+        "Bola de bronce pequeña",
+        "Zapatos de bronce impresionantes",
+        "Silla de acero a medida",
+        "Bike de goma impresionante",
+        "Jabón de plástico práctico",
+        "Tocino de granito increíble",
+        "Teclado de metal elegante",
+        "Salchichas de madera electrónica",
+        "Sabrosos guantes de madera",
+        "Pizza de hormigón sabrosa",
+        "Lujoso queso de metal",
+        "Guantes de goma impresionantes",
+        "Coche suave y elegante",
+        "Ensalada fresca con licencia",
+        "Toallas congeladas ergonómicas",
+        "Teclado de goma moderno",
+        "Pollo de plástico hecho a mano",
+        "Pollo de goma lujoso",
+        "Pescado suave práctico",
+        "Camisa de bronce ergonómica",
+        "Tocino de plástico artesanal",
+        "Pantalones de plástico sin marca",
+        "Salchichas de madera modernas",
+        "Zapatos de acero hechos a mano",
+        "Bicicleta de acero rústica",
+        "Hermosa ensalada congelada",
+        "Pollo bronce hecho a mano",
+        "Bicicleta de granito elegante",
+        "Salchichas de concreto genérico",
+        "Atún de plástico increíble",
+        "Queso fresco a medida",
+        "Tocino fresco electrónico",
+        "Coche de madera con licencia",
+        "Pescado reciclado",
+        "Guantes suaves prácticos",
+        "Chips suaves prácticos",
+        "Pollo pequeño y suave",
+        "Ratón fresco inteligente",
+        "Ratón de metal moderno",
+        "Sabrosos guantes de granito",
+        "Camisa de acero pequeña",
+        "Computadora de concreto refinada",
```

Comment:
"computadora" is from latino america, we might want to change to "ordenador" which is from spain. But maybe this isn't important :-)